ui/apps/platform/src/types/networkFlow.proto.ts

```diff
@@ -100,16 +100,17 @@ export type Node = {
     nonIsolatedIngress: boolean;
     nonIsolatedEgress: boolean;
     queryMatch: boolean;
-    outEdges: Edge[];
+    outEdges: OutEdges;
 };
 
-export type Edge = {
-    [key: string]: {
+export type OutEdges = Record<
+    string,
+    {
         properties: EdgeProperties[];
-    };
-};
+    }
+>;
```

Comment:
oh just curious why you decided to change this?

Reply:
Looks like I made a mistake here

Reply:
It was actually incorrect. `outEdges` is an object where the key is the node index and the value is an object with the field `properties`. I just fixed it here since the autosuggestion wasn't working due to wrong typing